### ISSUE TITLE
Trained weights

### ISSUE THREAD

**Issue by mayug** [yassouali/SCL#3]
Hi, 

Great work. I am interested in using spatial contrastive loss as a pretraining stage in my work. Could you please upload the pre-trained weights for Resnet12?

Thanks and regards
Mayug

**Comment by yassouali** [comment]
HI @mayug 


I tried finding the pretrained weight but unfortunately, i am not able to find them, however, the run should be quite quick and will only take a few hours, please see [run.sh](https://github.com/yassouali/SCL/blob/master/scripts/run.sh) to get the pretrained model.

Best.
